src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((style) => style.trim())
+    .reduce((stylesObject, style) => {
```

Comment:
Here you can add filtering of empty lines.
```suggestion
    .map((style) => style.trim())
    .filter(style => style)
    .reduce((stylesObject, style) => {
```